colcon-test/action.yaml

```diff
@@ -0,0 +1,138 @@
+name: colcon-test
+description: ""
+
+inputs:
+  rosdistro:
+    description: ""
+    required: true
+  target-packages:
+    description: ""
+    required: true
+  build-depends-repos:
+    description: ""
+    required: false
+  label:
+    description: ""
+    required: false
+    default: ".*" # Run all tests
```

Comment:
Could you explain why this `.*` works correctly?

Reply:
I guess `files: lcov_${{ inputs.label }}/total_coverage.info` would be an invalid path.

Reply:
Oh... I will update direcory name.

Reply:
> Could you explain why this `.*` works correctly?

The `label` argument accepts regex. So I use `.*` to run all tests.

Reply:
Then, it sounds like we can support multiple labels, not one.

Reply:
> Then, it sounds like we can support multiple labels, not one.

Yes, if you regex, we can support multiple labels, not one.

Reply:
> I guess `files: lcov_${{ inputs.label }}/total_coverage.info` would be an invalid path.

I modified to use default directory name. Since I set `zero-counter` option for each run, we can reuse same directory.
634f4f2